internal/api/user_test.go

```diff
@@ -326,3 +326,54 @@ func TestDeleteUser(t *testing.T) {
 
 	assert.Equal(t, http.StatusOK, w.Code)
 }
+
```

Comment:
I sort the tests by when the function is exited (1. Unauthenticated, 2. FormError, 3. Wrong Password, 4. Storage Error, 5. Ok)

The case for Unauthenticated, StorageError and FormError is not covered yet.